### PR TITLE
improvements [code-style, syntax, reusal]

### PR DESCRIPTION
**Pull request by jay-bricksoft**

### CODE REVIEW

File: src/Particles.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default class Particles extends PureComponent<ParticlesProps, ParticlesSt
 	}
 	
 	componentWillUnmount(){
-		this.state.library.destroy();
+		this.destroy();
```
Comment (jay-bricksoft) [inline src/Particles.tsx]: we have a method for this - use it

File: src/lib/getDefaultParams.ts [hunk 2]
```diff
@@ -1,120 +1,118 @@
...
-				number: {
-					value: 40,
-					density: {
-						enable: false,
-						value_area: 1200
-					}
-				},
-				color: {
-					value: '#FFF'
+	() => ({
```
Comment (jay-bricksoft) [inline src/lib/getDefaultParams.ts]: we dont do anything else but asignment -> return object literal instantly

File: src/lib/index.ts [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
-export * from './IParams';
-export * from './Utils';
 import Interact from './Interact';
 import Modes from './Modes';
 import Particle from './Particle';
 import ParticleManager from './ParticleManager';
 import ParticlesLibrary from './ParticlesLibrary';
 import Vendors from './Vendors';
 
+export * from './getDefaultParams';
```
Comment (jay-bricksoft) [inline src/lib/index.ts]: exports should come after imports, regardless if they also import

**Comment by Wufe** [comment]
Thanks for your contribution

**Pull request merged**